ansible/roles/kafka-logging-setup/files/kafka-lv-zookeepers.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: "local.storage.openshift.io/v1"
```

Comment:
Ditto on this file too. (We should delete this file since it won't be in use anymore.)

Reply:
deleted